src/amp/components/StickyAd.tsx

```diff
@@ -2,6 +2,21 @@ import React from 'react';
 import { Ad } from './Ad';
 import type { AdProps } from './Ad';
 
+// This CSS is added to global styles in document.tsx, to add the Advertisment label to the sticky
+export const stickyAdLabel = `
```

Comment:
I think this could be a tad more explicit.

```suggestion
export const stickyAdLabelCss = `
```